src/dali_backend.cc

```diff
@@ -26,9 +26,43 @@
 
 #include "triton/backend/backend_model.h"
 #include "triton/backend/backend_model_instance.h"
+#include "src/dali_executor/utils/utils.h"
 
 namespace triton { namespace backend { namespace dali {
 
+struct ModelParameters {
+  explicit ModelParameters(common::TritonJson::Value &model_config)
+  {
+    common::TritonJson::Value params;
+    model_config.MemberAsObject("parameters", &params);
+    GetMember(params, num_threads_key, &num_threads);
+  }
+
+ private:
+  template <typename T>
+  void GetMember(
+      common::TritonJson::Value &params,
+      const char* key,
+      T *value)
```

Comment:
```suggestion
      T &value)
```
?

Reply:
done.
I wanted to make the signature similar to the GetMemberAs... from triton API, but there is no need for that actually.